Multiple playlists cleaning, stream fixes

BellezaEmporium opened the pull request:
This PR cleans some of the duplicates that are on the first playlists, and the International playlist (I would also need an explanation about why using the term "International" in fact.)

*Resume*
- Cleaned duplicate streams on some playlists, still ongoing.
- Renamed certain streams because their first name were either incorrect, or not meaning much.
- Fixed some links for certain streams.
- Use the playlist.m3u8 instead of chunklist.m3u8. I tend to think those are better for people with bandwidth issues (as far as I know, the playlist contains all the available chunklists that he can deal with, looking at the bandwidth).
- Added a few channels, on certain countries, when available.
- Used the new (and discussed) naming convention for some channels, when useful.

P.S : I explained what I've done also on each commit, excluding some, where no big modifications were made.

Next PR will be sorting some channels from the unsorted playlist.

BellezaEmporium committed:
Deleting duplicates on "ad.m3u"

BellezaEmporium committed:
Clean AE.m3u

- Deleted duplicates
- Prefer the use of playlists instead of chunklists, this will permit low-bandwidth users to get their stream correctly
- Modified Toheed TV stream link (Towheed TV on Google and their website)

BellezaEmporium committed:
Modify af.m3u

- Use playlists instead of chunklists for low-bandwidth users
- Removed duplicates
- Modified some links and appelations (US could be misleading... use Usbeki)

BellezaEmporium committed:
Clean "int.m3u"

- Removed multiple references to Jasmin TV (same channel but different chunklists...)
- Reorganized RedTraffic channels so that the user knows which channel it is (numbered channels mean nothing.)
- AST channels are numbered by their quality.

LaneSh4d0w commented:
P.S : I explained what I've done also on each commit, excluding the first one.

BellezaEmporium committed:
AM : Specify audio tracks for H1

BellezaEmporium committed:
AO : Add TV Zimbo

BellezaEmporium committed:
AO : Fix TV Zimbo

BellezaEmporium committed:
AT : Add all available ORF channels.

BellezaEmporium committed:
AT : Change Chunklist extensions to Playlist

freearhey commented:
@LaneSh4d0w As for the "International" playlist: all channels are sorted by country from which they are broadcast (this info can be found at [lyngsat.com](https://www.lyngsat.com/)), but some channels cannot be assigned to any country (such as "Red Bull TV" or "Jasmin TV") and for such cases the International playlist was created.

LaneSh4d0w commented:
Closing this for the moment, waiting for @freearhey to finish the link verification. Once he's done, i'll pull the latest m3u files (normally, no conflict) and i'll continue.

BellezaEmporium committed:
Merge branch 'remove-broken-links' of https://github.com/iptv-org/iptv

LaneSh4d0w commented:
I got the latest changes you've done. Reopening.

BellezaEmporium committed:
INT : Re-establishing a Jasmin TV link

- For SD and HD defs.
- Adding more data to Red Bull channel names.

philiprgbel commented:
Could you do that for me please thank yo

LaneSh4d0w commented:
> Could you do that for me please thank yo 

If it's what I understand, simply grab the code from my forked repository and you will get all the needed changes we're doing.

freearhey committed:
Merge branch 'master' into master

link8k commented on channels/ae.m3u:
```diff
@@ -13,16 +13,12 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 #EXTINF:-1 tvg-id="Al Arabiya ARB" tvg-name="Al Arabiya ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/7OrfyOx.jpg" group-title="News",Al Arabiya
 https://live.alarabiya.net/alarabiapublish/alarabiya.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Al Arabiya ARB" tvg-name="Al Arabiya ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/7OrfyOx.jpg" group-title="News",Al Arabiya
-https://shls-alarabiya-prod-dub.shahid.net/out/v1/f5f319206ed740f9a831f2097c2ead23/index.m3u8
```
This mirror works and is higher quality than https://live.alarabiya.net/alarabiapublish/alarabiya.smil/playlist.m3u8

LaneSh4d0w replied:
Ah ! When I had tested it it was worse, maybe I shall keep this one.

link8k commented on channels/ae.m3u:
```diff
@@ -13,16 +13,12 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 #EXTINF:-1 tvg-id="Al Arabiya Al Hadath ARB" tvg-name="Al Arabiya Al Hadath ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/rLR5q2x.png" group-title="News",Al Arabiya Al Hadath
 https://live.alarabiya.net/alarabiapublish/alhadath.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Al Arabiya Al Hadath ARB" tvg-name="Al Arabiya Al Hadath ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/rLR5q2x.png" group-title="News",Al Arabiya Al Hadath
-https://shls-hadath-prod-dub.shahid.net/out/v1/0e1a306399c346faac4226aa0858f99b/index.m3u8
```
Also works

LaneSh4d0w replied:
Noted.

BellezaEmporium committed:
AE : Kept the Shahid streams

Those streams were indicated to be better than the original streams. Added definitions.

BellezaEmporium committed:
AE : Mistakes were made

Weird issue.

BellezaEmporium reviewed (commented):
Ready for review (collaborators). Moved to draft because I'll still be working on it. Real-time reviews are more than welcome.

themak1985 commented:
Best to do these in chunks - will review when you mark as ready for review.

LaneSh4d0w commented:
> Best to do these in chunks - will review when you mark as ready for review.

Yes, I suppose i'll do it part by part. I'll make some more playlists analysis and then i'll move the draft as RtR

BellezaEmporium committed:
Merge branch 'master' of https://github.com/iptv-org/iptv
Get latest changes from upstream, to prevent anomalies

BellezaEmporium committed:
Merge branch 'master' of https://github.com/iptv-org/iptv
Get latest changes from upstream, to prevent conflicts happening.

link8k commented on channels/ae.m3u:
```diff
@@ -13,15 +13,15 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
+#EXTINF:-1 tvg-id="Al Aan TV ARB" tvg-name="Al Aan TV" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6WiXPd9.png" group-title="",Al Aan TV
 https://streamingcdn.alaan.tv/Publisher/alaantv_twitter/playlist.m3u8
-#EXTINF:-1 tvg-id="Al Arabiya ARB" tvg-name="Al Arabiya ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/7OrfyOx.jpg" group-title="News",Al Arabiya
+#EXTINF:-1 tvg-id="Al Arabiya ARB" tvg-name="Al Arabiya ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/7OrfyOx.jpg" group-title="News",Al Arabiya (SD)
```
I don't think you need the quality here as this url adapts from SD to HD

BellezaEmporium committed:
AE : Reflect adaptive over SD

BellezaEmporium committed:
Merge branch 'master' of https://github.com/iptv-org/iptv
Get updated with upstream

BellezaEmporium committed:
AO : Fix TV Zimbo

BellezaEmporium committed:
AM : Deleted Tuntunik (403 Forbidden)

Will be put back again if found to be a non 24-7 channel, in which I cannot confirm.

BellezaEmporium committed:
AR : Removed dupes, added group names

- Lots of Canal 26 links for nothing, added definitions per URL queries.
- Cleaned several others.
- Added "Local" for many channels.

BellezaEmporium committed:
Merge branch 'master' of https://github.com/LaneSh4d0w/iptv

BellezaEmporium committed:
AR : Set Adaptive on Canal 10.

BellezaEmporium committed:
AU : Deleted a dupe

Reason : Site is the same

BellezaEmporium committed:
Merge branch 'master' of https://github.com/LaneSh4d0w/iptv

LaneSh4d0w commented:
First batch done. R2R

themak1985 commented on channels/ad.m3u:
```diff
@@ -1,5 +1,3 @@
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Catalan" tvg-logo="https://i.imgur.com/kJCjeQ4.png" group-title="",RTVA
-http://videos.rtva.ad:1935/live/rtva/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Catalan" tvg-logo="https://i.imgur.com/kJCjeQ4.png" group-title="",RTVA
-http://videos.rtva.ad:1935/live/web/playlist.m3u8
```
I would consider keeping one of these as a backup.

LaneSh4d0w replied:
Noted.

themak1985 commented on channels/ae.m3u:
```diff
@@ -13,16 +13,12 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 #EXTINF:-1 tvg-id="Abu Dhabi TV ARB" tvg-name="Abu Dhabi 1 ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oExmYqk.jpg" group-title="General",Abu Dhabi TV
 https://admdn2.cdn.mangomolo.com/adtv/smil:adtv.stream.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6WiXPd9.png" group-title="",Al Aan TV
+#EXTINF:-1 tvg-id="Al Aan TV ARB" tvg-name="Al Aan TV" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6WiXPd9.png" group-title="",Al Aan TV
```
tvg-id and tvg-name should both be `Al Aan TV`

LaneSh4d0w replied:
Noted.

themak1985 commented on channels/ae.m3u:
```diff
@@ -13,16 +13,12 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 #EXTINF:-1 tvg-id="Al Arabiya ARB" tvg-name="Al Arabiya ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/7OrfyOx.jpg" group-title="News",Al Arabiya
 https://live.alarabiya.net/alarabiapublish/alarabiya.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Al Arabiya ARB" tvg-name="Al Arabiya ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/7OrfyOx.jpg" group-title="News",Al Arabiya
-https://shls-alarabiya-prod-dub.shahid.net/out/v1/f5f319206ed740f9a831f2097c2ead23/index.m3u8
```
This should stay

LaneSh4d0w replied:
I remember you telling me about those Shahid links, I forgot that one ? Oops. Noted.

LaneSh4d0w replied:
It seems you were looking at an outdated version, both links are there.

themak1985 commented on channels/ae.m3u:
```diff
@@ -13,16 +13,12 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 #EXTINF:-1 tvg-id="Al Arabiya Al Hadath ARB" tvg-name="Al Arabiya Al Hadath ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/rLR5q2x.png" group-title="News",Al Arabiya Al Hadath
 https://live.alarabiya.net/alarabiapublish/alhadath.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Al Arabiya Al Hadath ARB" tvg-name="Al Arabiya Al Hadath ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/rLR5q2x.png" group-title="News",Al Arabiya Al Hadath
-https://shls-hadath-prod-dub.shahid.net/out/v1/0e1a306399c346faac4226aa0858f99b/index.m3u8
```
This should stay

themak1985 commented on channels/ae.m3u:
```diff
@@ -34,10 +30,10 @@ https://citrusstv.akamaized.net/hls/live/687285/CTV/index.m3u8
-https://dmithrvll.cdn.mangomolo.com/dubairacing/smil:dubairacing.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eytrf0Y.png" group-title="Sport",Dubai Racing 3
+https://dmisvthvll.cdn.mangomolo.com/events/smil:events.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Dubai Racing 2 ARB" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eytrf0Y.png" group-title="Sport",Dubai Racing 2
```
tvg-id should be blank, not in EPG

LaneSh4d0w replied:
Have written it in the wrong place. Noted.

themak1985 commented on channels/ae.m3u:
```diff
@@ -34,10 +30,10 @@ https://citrusstv.akamaized.net/hls/live/687285/CTV/index.m3u8
+https://dmisvthvll.cdn.mangomolo.com/events/smil:events.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Dubai Racing 2 ARB" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eytrf0Y.png" group-title="Sport",Dubai Racing 2
+https://dmithrvll.cdn.mangomolo.com/dubairacing/smil:dubairacing.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Dubai Racing 3 ARB" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eytrf0Y.png" group-title="Sport",Dubai Racing 3
```
tvg-id and tvg-name should be `Dubai Racing 3`

themak1985 commented on channels/ae.m3u:
```diff
@@ -55,9 +51,9 @@ https://admdn3.cdn.mangomolo.com/emarat/smil:emarat.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/9Bun2IX.png" group-title="",Majid TV
 https://admdn4.cdn.mangomolo.com/majid/smil:majid.stream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="MBC 1 ARB" tvg-name="MBC 1 ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eKBXOKd.png" group-title="",MBC
+#EXTINF:-1 tvg-id="MBC 1 ARB" tvg-name="MBC 1 ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eKBXOKd.png" group-title="",MBC 1
```
The channel name is `MBC`, the tvg-id and tvg-name should be `MBC`

LaneSh4d0w replied:
Noted

themak1985 commented on channels/ae.m3u:
```diff
@@ -55,9 +51,9 @@ https://admdn3.cdn.mangomolo.com/emarat/smil:emarat.stream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="MBC 1 ARB" tvg-name="MBC 1 ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eKBXOKd.png" group-title="",MBC 1
 https://shls-mbc1ksa-prod-dub.shahid.net/out/v1/451b666db1fb41c7a4bbecf7b4865107/index.m3u8
-#EXTINF:-1 tvg-id="MBC" tvg-name="MBC 2 ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/KXuzL1u.png" group-title="Movies",MBC 2 HD
+#EXTINF:-1 tvg-id="MBC" tvg-name="MBC 2 ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/KXuzL1u.png" group-title="Movies",MBC 2
```
tvg-id and tvg-name should be `MBC 2`

themak1985 commented on channels/ae.m3u:
```diff
@@ -93,16 +89,16 @@ https://dmiffthvll.cdn.mangomolo.com/noordubaitv/smil:noordubaitv.smil/chunklist
 http://peacetv.api.visionip.tv/live/ASHTTP/peacetv-peacetv-peacetv-english-hsslive-25f-16x9-MB/playlist.m3u8
 #EXTINF:-1 tvg-id="Sama Dubai ARB" tvg-name="Sama Dubai ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/bF6I3N1.jpg" group-title="",Sama Dubai
 https://dmieigthvll.cdn.mangomolo.com/samadubai/smil:samadubai.stream.smil/chunklist.m3u8
+#EXTINF:-1 tvg-id="Sharjah TV ARB" tvg-name="Sharjah TV ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/FAp0RtO.png" group-title="",Sharjah TV
```
tvg-id and tvg-name should be `Sharjah TV`

themak1985 commented on channels/af.m3u:
```diff
@@ -1,21 +1,15 @@
-https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV US
-https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Usbeki" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV (Usbeki)
```
Keep channel name as Ariana TV, keep language as Pushto.

LaneSh4d0w replied:
The language used is mentioned on Ariana TV's website. The US mention is Usbeki, not Pushto.

themak1985 replied:
Usbeki is not an iso 639-3 language, Uzbek is. However I just listened to the stream, its definitely broadcasting in Pushto.

themak1985 commented on channels/int.m3u:
```diff
@@ -5,66 +5,32 @@ http://51.158.190.97/hls/stream.m3u8
 https://live.creacast.com/cna/smil:cna.smil/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/tXTjWgP.png" group-title="Religious",Christian Youth Channel
 http://media.smc-host.com:1935/cycnow.com/cyc2/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/XQ2vMHd.png" group-title="Music",DJing
```
All these channels should stay

LaneSh4d0w replied:
Those channels aren't livestreams, they all redirect to a 48 second stream of an advertisment for "Hot Hot Hot", hence the deletion of all channels.

themak1985 commented on channels/am.m3u:
```diff
@@ -1,9 +1,9 @@
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/3vQVwUV.jpg" group-title="",Amga
 https://streamer1.connectto.com/AMGA_WEB_1202/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/MKVkDVo.png" group-title="",H1
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/MKVkDVo.png" group-title="",H1 (Audio Track 1)
```
The channel names should only be the channel.

LaneSh4d0w replied:
Thing is there's something linked to the audio tracks in the stream. Shall I delete the mention and count the second one as a backup ?

themak1985 replied:
that works.

themak1985 commented on channels/am.m3u:
```diff
@@ -1,9 +1,9 @@
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/MKVkDVo.png" group-title="",H1 (Audio Track 1)
 http://amtv1.livestreamingcdn.com/am2abr/tracks-v1a1/index.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/MKVkDVo.png" group-title="",H1
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/MKVkDVo.png" group-title="",H1 (Audio Track 2)
```
The channel names should only be the channel.

themak1985 commented on channels/ao.m3u:
```diff
@@ -1,3 +1,5 @@
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/12aPxO7.png" group-title="General",TPA1
 https://api.new.livestream.com/accounts/1181452/events/8865379/live.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwng5WHhHdPIDydhUd-TmZXIxqvYqzTmPtCdDtDnmCw=s900-c-k-c0x00ffffff-no-rj" group-title="General",TV Zimbo
+https://api.new.livestream.com/accounts/29819666/events/9254936/live.m3u8
```
Broken Link

themak1985 commented on channels/ao.m3u:
```diff
@@ -2,4 +2,4 @@
 https://api.new.livestream.com/accounts/1181452/events/8865379/live.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwng5WHhHdPIDydhUd-TmZXIxqvYqzTmPtCdDtDnmCw=s900-c-k-c0x00ffffff-no-rj" group-title="General",TV Zimbo
-https://api.new.livestream.com/accounts/29819666/events/9254936/live.m3u8
+https://api.new.livestream.com/accounts/29819666/events/9254936/broadcasts/215608865.secure.m3u8
```
Link is broken

LaneSh4d0w replied:
Outdated version of my PR + it tends to break, maybe a timestamped stream.

themak1985 commented on channels/at.m3u:
```diff
@@ -11,20 +11,24 @@ http://185.85.28.19/oe3sd/orf.sdp/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/Wu6SMvb.png" group-title="",R9
 https://ms01.w24.at/R9/smil:liveeventR9.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/mw9GNmh.png" group-title="",RTL Austria
-http://188.35.9.5:10020/stolen/502r/hot.dll
```
Working Link - i believe they are part time.

LaneSh4d0w replied:
Ah ! The link fooled me for a moment.

themak1985 commented on channels/ae.m3u:
```diff
@@ -97,8 +97,11 @@ https://svs.itworkscdn.net/smc2live/smc2tv.smil/playlist.m3u8
 https://svs.itworkscdn.net/smc4sportslive/smc4.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="Sharqiya from Kalba ARB" tvg-name="Sharqiya from Kalba ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6Er1lh6.png" group-title="",Sharqiya from Kalba TV
 https://svs.itworkscdn.net/kablatvlive/kabtv1.smil/playlist.m3u8
+<<<<<<< HEAD
```
What is this?

LaneSh4d0w replied:
That was my GitHub finding an anomaly, you're looking at an outdated version of my PR.

themak1985 commented on channels/ae.m3u:
```diff
@@ -97,8 +97,11 @@ https://svs.itworkscdn.net/smc2live/smc2tv.smil/playlist.m3u8
+<<<<<<< HEAD
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/nsxrqta.jpg" group-title="",Towheed TV
 http://51.210.199.1/hls/stream.m3u8
+=======
```
?

LaneSh4d0w replied:
That was my Git finding an anomaly, you're looking at an outdated version of my PR.

themak1985 commented on channels/ae.m3u:
```diff
@@ -97,8 +97,11 @@ https://svs.itworkscdn.net/smc2live/smc2tv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/nsxrqta.jpg" group-title="",Towheed TV
 http://51.210.199.1/hls/stream.m3u8
+=======
+>>>>>>> 8de93c697e21852e05e38eb2f320d65f69b63f1a
```
?

LaneSh4d0w replied:
That was my Git finding an anomaly, you're looking at an outdated version of my PR.

LaneSh4d0w commented:
Noted for all, I'll make changes in a few hours.

LaneSh4d0w commented on channels/ao.m3u:
```diff
@@ -2,4 +2,4 @@
 https://api.new.livestream.com/accounts/1181452/events/8865379/live.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwng5WHhHdPIDydhUd-TmZXIxqvYqzTmPtCdDtDnmCw=s900-c-k-c0x00ffffff-no-rj" group-title="General",TV Zimbo
-https://api.new.livestream.com/accounts/29819666/events/9254936/broadcasts/215608865.secure.m3u8
+http://api.new.livestream.com/accounts/29819666/events/9254936/broadcasts/215814388.m3u8
```
It tends to be broken everytime I suppose (probable timestamp on stream). The "live.m3u8" doesn't exist in this livestream.

I think I'll remove it from the time being, until a better alternative link comes out.

LaneSh4d0w commented on channels/ar.m3u:
```diff
@@ -17,65 +15,49 @@ http://45.6.4.50:50601/America_Tucuman/video.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/dKpLbh6.png" group-title="Music",Music Top (HD 720P)
 http://live-edge01.telecentro.net.ar:1935/live/msctphd-720/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/dKpLbh6.png" group-title="Music",Music Top
-https://live-edge01.telecentro.net.ar/live/msctphd-720/playlist.m3u8
```
I deleted it because this link is the same as the one above, only the port has been added.

LaneSh4d0w commented on channels/ar.m3u:
```diff
@@ -17,65 +15,49 @@ http://45.6.4.50:50601/America_Tucuman/video.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/Op0zdh5.jpg" group-title="Local",Fenix TV (Ciudad de La Rioja)
 https://stmv1.questreaming.com/fenixlarioja/fenixlarioja/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/cd0hP5e.png" group-title="",Garage TV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/cd0hP5e.png" group-title="Lifestyle;Sports",Garage TV
```
Well for this one, it depends... Sometimes you got moto races, sometimes there's some shows about automobile reparations.

BellezaEmporium committed:
AD : Keep the other RTVA link as backup.

BellezaEmporium committed:
AE : Change some identifications

BellezaEmporium committed:
AE : Change some identifications.

BellezaEmporium committed:
AE : Rename MBC

BellezaEmporium committed:
AE : Rename ID and names accordingly.

BellezaEmporium committed:
AO : Delete TV Zimbo

BellezaEmporium committed:
AR : Make Garage TV count as a Sports channel.

BellezaEmporium reviewed (commented):
Reviewed.

BellezaEmporium committed:
AT : Keep RTL Austria

BellezaEmporium reviewed (commented):
Reviewed.

themak1985 commented on channels/au.m3u:
```diff
@@ -5,8 +5,6 @@ https://i.mjh.nz/au/Melbourne/tv.acctv.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/VsvBUCa.jpg" group-title="Business",Bloomberg TV Australia
 https://liveprodapnortheast.global.ssl.fastly.net/btv/desktop/aus_live.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/VsvBUCa.jpg" group-title="Business",Bloomberg TV Australia
-https://liveprodapnortheast.global.ssl.fastly.net/oz1/Channel-OZTVqvs-AWS-tokyo-1/Source-OZTVqvs-1000-1_live.m3u8
```
Keep this link.

BellezaEmporium committed:
AF : Redefine Ariana TV

BellezaEmporium committed:
AM : Count the second H1 as a backup.

BellezaEmporium committed:
AU : Keep the second link as a backup.

BellezaEmporium reviewed (commented):
Ready for a second review.

BellezaEmporium committed:
Merge branch 'master' of https://github.com/iptv-org/iptv
Keep updated with upstream

BellezaEmporium committed:
Merge branch 'master' of https://github.com/LaneSh4d0w/iptv
